src/control/server/ctl_storage_rpc_test.go

```diff
@@ -345,6 +346,8 @@ func TestServer_CtlSvc_StorageScan_PostIOStart(t *testing.T) {
 		}
 		bioHealthResp.Model = ctrlr.Model
 		bioHealthResp.Serial = ctrlr.Serial
+		bioHealthResp.TotalBytes = uint64(idx) * uint64(humanize.TByte)
+		bioHealthResp.AvailBytes = uint64(idx) * uint64(humanize.TByte)
```

Comment:
Might be good to make these values different so it's clear in the tests that they came from the right fields

Reply:
fixed